### PR TITLE
docs: update logs.md

### PR DESCRIPTION
**Pull request by manojs1978**
### Prepare

- [ ] Read [PR guidelines](https://github.com/JanssenProject/jans/blob/main/docs/CONTRIBUTING.md#prs)
- [ ] Read [license information](https://github.com/JanssenProject/jans/blob/main/LICENSE)

-------------------

#### Target issue
  
  
  
  
closes #issue-number-here

#### Implementation Details
  

-------------------
### Test and Document the changes
- [ ] Static code analysis has been run locally and issues have been fixed
- [ ] Relevant unit and integration tests have been added/updated
- [ ] Relevant documentation has been updated if any (i.e. user guides, installation and configuration guides, technical design docs etc)

### CODE REVIEW

**Review by mo-auto — APPROVED** [review]

**Review by mo-auto — DISMISSED** [review]

File: docs/admin/scim/logs.md [hunk 1]
```diff
@@ -2,16 +2,21 @@
...
-The Janssen Project documentation is currently in development. Topic pages are being created in order of broadest relevance, and this page is coming in the near future.
-
+## SCIM Logs
+    Jans Server logs usually reveal the source of problems when things are going wrong: the first place to look is the SCIM log.  
+   Authorization issues (access tokens problems, for instance) are on the side of oxAuth (the authorization server)
+        • SCIM log is located at  /opt/jans/jetty/jans-scim/logs/scim.log
+        • Auth log is at /opt/jans/jetty/jans-auth/logs/jans-auth.log
+        • If using the SCIM custom script in order to intercept API calls and apply custom logic, the script log is also        
+          useful: /opt/jans/jetty/jans-scim/logs/scim_script.log
+   Generally it's convenient to set the logging level for both jans-Auth and SCIM to DEBUG.
```
Comment (ossdhaval) [inline docs/admin/scim/logs.md]: `Generally it's convenient to set the logging level for both jans-Auth and SCIM to DEBUG.`

This statement is too broad. Either remove it or add link to where steps for enabling debug are listed. Other content look ok.

File: docs/admin/scim/logs.md [hunk 2]
```diff
@@ -2,16 +2,21 @@
...
 ## Have questions in the meantime?
```
Comment (ossdhaval) [inline docs/admin/scim/logs.md]: Consider removing this section.

**Review by ossdhaval — CHANGES_REQUESTED** [review]

File: docs/admin/scim/logs.md (hunk 2 above)
Comment (manojs1978) [reply docs/admin/scim/logs.md]: made changes accordingly

**Review by ossdhaval — APPROVED** [review]

**Comment by manojs1978** [comment]
very old updates , which are already updated

**Pull request closed without merging**